nixos/modules/tasks/filesystems.nix

```diff
@@ -301,8 +301,20 @@ in
             unitConfig.DefaultDependencies = false; # needed to prevent a cycle
             serviceConfig.Type = "oneshot";
           };
-
-      in listToAttrs (map formatDevice (filter (fs: fs.autoFormat) fileSystems));
+      in listToAttrs (map formatDevice (filter (fs: fs.autoFormat) fileSystems)) // {
+        # Mount /sys/fs/pstore for evacuating panic logs and crashdumps from persistent storage onto the disk using systemd-pstore.
+        # This cannot be done with the other special filesystems because the pstore module is not loaded at that point.
```

Comment:
Can you add another comment line explaining this is usually set up by systemd in initrd, so we know we can remove it?

Reply:
Frankly I don't understand enough about how and why special filesystems are done the way they are in NixOS so I'm not sure what kind of explanatory comment to add that wouldn't be possibly misleading. I think the current comment indicates well enough that it is like the other special filesystems, and save for the mentioned exception it should be handled the same way.